docs/application-connector/10-02-application-gateway-troubleshooting.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Error when calling a registered Service
```

Comment:
This document should be structured differently. 

Error -> check logs -> no logs? -> deregister service -> register service (why? shouldn't you let the user know that it will recreate the access service?) -> still error -> check URL